src/screens/WalletScreen/List/index.js

```diff
@@ -10,6 +10,7 @@ import Images from '../../../global/AssetsImages';
 import WalletCard from '../../../components/WalletCard';
 import BackgroundImage from '../../../components/common/BackgroundImage';
 import FakeNavigationBar from '../../../components/common/FakeNavigationBar';
+import { formatETH } from '../../../utils/formatters/amountFormatter';

```

Comment:
Can you remove this util? I created one in the `utils/formatters.js` that should be used instead of this.